tests/align/talign.nim

```diff
@@ -51,3 +51,26 @@ type Bug[T] = object
 
 var bug: Bug[int]
 doAssert sizeof(bug) == 128, "Oops my size is " & $sizeof(bug) # 16
+
+
+import std/asyncdispatch
+
+block:
+  type
+    ValidatorPubKey = object
+      blob: array[96, byte]
+
+  proc f() {.async.} =
+    var x {.align: 16.}: uint8
```

Comment:
this align doesn't matter for the purpose of the bug

Reply:
Yeah, It's harmless, though.